src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java

```diff
@@ -169,11 +194,19 @@ public final class SiteUtil {
     private static final Map<String, String> SINCE_VERSION_FOR_INHERITED_PROPERTY = Map.ofEntries(
         Map.entry("MissingDeprecatedCheck.violateExecutionOnNonTightHtml", V824),
         Map.entry("NonEmptyAtclauseDescriptionCheck.violateExecutionOnNonTightHtml", "8.3"),
+        Map.entry("HeaderCheck.charset", V50),
+        Map.entry("HeaderCheck.fileExtensions", V69),
+        Map.entry("HeaderCheck.headerFile", V32),
+        Map.entry(HEADERCHECK_HEADER, V50),
```

Comment:
Translation of `HeaderCheck` to upper snake case is `HEADER_CHECK`, please do for all.

Reply:
done